linalg/sundials.cpp

```diff
@@ -1318,6 +1315,13 @@ void KINSolver::Mult(Vector &x,
       MFEM_VERIFY(NV_LOCLENGTH_P(y) == x.Size(), "");
       NV_DATA_P(y_scale) = x_scale.GetData();
       NV_DATA_P(f_scale) = fx_scale.GetData();
+
+      int rank;
+      MPI_Comm_rank(NV_COMM_P(y), &rank);
+      if (rank == 0) {
+         flag = KINSetPrintLevel(sundials_mem, print_level);
+         MFEM_VERIFY(flag == KIN_SUCCESS, "KINSetPrintLevel() failed!");
```

Comment:
Should also have
```
flag = KINSetPrintLevel(sundials_mem, print_level);
MFEM_VERIFY(flag == KIN_SUCCESS, "KINSetPrintLevel() failed!");
```
in the `if (!Parallel())` block for anyone running in serial.

Reply:
Missed that, added.